btree/btree.c

```diff
@@ -4337,30 +4335,30 @@ static void vkvv_fid(const struct nd *node, struct m0_fid *fid)
 
 }
 
-static uint16_t vkvv_lnode_rec_key_size(const struct nd *node, int idx)
+static uint32_t vkvv_lnode_rec_key_size(const struct nd *node, int idx)
 {
 	struct vkvv_head *h   = vkvv_data(node);
-	struct dir_rec   *rec = (void *)h +sizeof(*h) + h->vkvv_dir_offset;
+	struct dir_rec   *rec = (void *)h + sizeof(*h) + h->vkvv_dir_offset;
 
 	return rec[idx+1].key_offset - rec[idx].key_offset;
 }
 
-static uint16_t vkvv_inode_rec_key_size(const struct nd *node, int idx)
+static uint32_t vkvv_inode_rec_key_size(const struct nd *node, int idx)
 {
-	uint16_t *offset;
-	uint16_t *prev_offset;
+	uint32_t *offset;
+	uint32_t *prev_offset;
 
 	if (idx == 0){
-		offset = vkvv_get_key_offset(node,idx);
+		offset = vkvv_get_key_offset(node, idx);
 		return *offset;
 	} else {
-		offset      = vkvv_get_key_offset(node,idx);
-		prev_offset = vkvv_get_key_offset(node,idx-1);
+		offset      = vkvv_get_key_offset(node, idx);
+		prev_offset = vkvv_get_key_offset(node, idx - 1);
 		return (*offset - *prev_offset);
 	}
 }
 
-static uint16_t vkvv_rec_key_size(const struct nd *node, int idx)
+static uint32_t vkvv_rec_key_size(const struct nd *node, int idx)
 {
 	/**
 	 * @brief This function is used to get the size of the key at the given
```

Comment:
Please move function description outside of the function body (here and at all the other places.)

Reply:
Done